lambda/writePass/index.js

```diff
@@ -20,7 +21,7 @@ const { generateRegistrationNumber } = require('../captchaUtil');
 
 // default opening/closing hours in 24h time
 const DEFAULT_AM_OPENING_HOUR = 7;
-const DEFAULT_PM_OPENING_HOUR = 13;
+const DEFAULT_PM_OPENING_HOUR = PM_ACTIVATION_HOUR;
```

Comment:
Instead of creating a new variable here `DEFAULT_PM_OPENING_HOUR` which is used only once down below, I would just remove it and use `PM_ACTIVATION_HOUR` instead.  I might even name the constant `DEFAULT_PM_OPENING_HOUR` inside of the dynamoUtil.

Reply:
A few more files were edited to reflect changing the constant name in dynamoUtil. Addressed in: https://github.com/bcgov/parks-reso-api/pull/332/commits/4fbb3593a5555e06dea57bc46e187becdba728e7